drivers/clock_control/clock_control_mcux_ccm.c

```diff
@@ -179,6 +218,13 @@ static int mcux_ccm_off(const struct device *dev,
 		CLOCK_DisableClock(sai_clocks[instance]);
 		return 0;
 #endif
+#if defined(CONFIG_SOC_MIMX8ML8_ADSP)
+	case IMX_CCM_SAI1_CLK:
```

Comment:
Same here you can group the code with `#if defined(CONFIG_SOC_MIMX8QM6_ADSP) || defined(CONFIG_SOC_MIMX8QX6_ADSP)` above.